.gitignore

```diff
@@ -1,9 +1,15 @@
 build/
 test/sandbox
 .DS_Store
+
```

Comment:
I think it is not good to add new things to the ignore file

Reply:
I can drop this, but swap files are pretty common on ignores.